process/predicting-milestones.md

```diff
@@ -38,34 +38,29 @@ This document is an informal guide to predicting the likely duration of Recommen
 
 The first set of milestones are drawn up before work starts, and put in an initial charter. In this phase, an idea is firmed up from "hey we could do this" to a fully fledged, chartered Working Group. If the work is entirely new, or would take an existing group in a radical new direction, the Process Document [requires](/policies/process/#WGCharterDevelopment) that Advance Notice be sent to the AC. Often, an initial draft charter is sent along with this notice (but this is not required). The aim here is to avoid surprises, to quickly gather brainstorming-type feedback, and to get buy-in on the proposed direction from likely members of the eventual Working Group.
 
-> 
-Advance Notice
+> Time: Advance Notice
 : There is no set period for Advance Notice, but four weeks is a typical minimum period.
 {:.time}
 
->
-Advance Notice
+>Tips: Advance Notice
 : When developing materials for and advance notice, try to strike a balance between fast (but overly sketchy and vague) and thorough (but slow, maybe over-detailed or over-constraining).
 >
   If work has been prepared beforehand in a Community group, or formalizes existing, industry standard practices, then an Advance Notice might be skipped.
  {:.tip}
 
 The results from Advance Notice may indicate broad consensus on the proposed direction (if so, proceed rapidly to a formal charter review). Or they may indicate disagreement on the best direction forward (in which case, iterate). Some commentors may want a fast development of an initial version, while others may want a fully fledged solution.
 
-> 
-Advance Notice Feedback
+> Tip: Advance Notice Feedback
 : If feedback indicates disagreement on scope, consider proposing both a fast initial version *and* a requirements gathering and drafting item for a full-featured second version. This will help avoid delays due to disagreements on scope.
```

Comment:
```suggestion
: If feedback indicates disagreement on scope, consider proposing both a fast initial version *and* a requirements gathering and drafting item for a full-featured second version. This can help avoid delays due to disagreements on scope.
```